OWNERS

```diff
@@ -5,6 +5,4 @@ approvers:
   - paveldournov
   - qimingj
 reviewers:
-  - vicaire
-  - IronPan
-  - qimingj
+  - texasmichelle
```

Comment:
Doesn't that mean that your review will be required for all check-ins?

Reply:
I think it does

Reply:
No, it means that I can mark PR's with `lgtm`, but not `approve`.

Reply:
Approvers can submit both

Reply:
PRs require both to merge

Reply:
Adding here means one could lgtm all PRs for the repo. 
@texasmichelle do you need root level or just some subfolder?

Reply:
Michelle, I would rather keep folks from the Kubeflow/pipeline core team at that level. We will reorganize the OWNER files later on to match best practices. Thanks.

Reply:
OK, that sounds reasonable.

Reply:
My preference is to move the pipelines examples into [kubeflow/examples](https://github.com/kubeflow/examples). The intention of this PR is to provide reviews for pipeline functionality, not just samples.

Only a subset of your team is in the file currently - adding the remainder would help with the sparsity issue. Happy to include them in this PR if you have GitHub IDs.

Reply:
@vicaire expertise is provided by approvers. PRs can't be merged until approvers approve.
Listing people as reviewers is a great way to get people involved and grow the pool of reviewers. You could do this either at the root directory or subdirectory. Which ever makes sense.

Here's the definition from our contributing guide

> Reviewers look for general code quality, correctness, sane software engineering, style, etc.

> Approvers look for holistic acceptance criteria, including dependencies with other features, forwards/backwards compatibility, API and flag definitions, etc

A good place to add reviewers would be places where non-Google expertise is needed. For example, it would be good to start getting non-Googlers involved in areas where changes are needed to run on other platforms. For example, areas related to persisting artifacts that need to be extended to support other object stores.

it would also be good to get non-googlers involved in reviewing reusable components/steps since we would like to get more folks contributing components.